test.js

```diff
@@ -238,6 +262,48 @@ describe('cli', function () {
     })
   })
 
+  describe('unknown host type', function () {
+    describe('with no context flag', function () {
+      it('should use default context', function () {
+        writePackageJson('1.0.1', {'repository': 'https://private-repo.company.org/private-owner/private-project.git'})
+
+        commit('feat: new context flag')
+        commit('fix: patch release')
+        execCli().code.should.equal(0)
+
+        var content = fs.readFileSync('CHANGELOG.md', 'utf-8')
+        content.should.include('](https://private-repo.company.org/private-owner/private-project/commits/')
+        content.should.match(/patch release/)
+        content.should.match(/new context flag/)
+        shell.exec('git tag').stdout.should.match(/1\.1\.0/)
+      })
+    })
+    describe('with context flag', function () {
```

Comment:
Should we also include a test that defines the context file within package.json itself? i.e. A test identical to `should use context keywords with a valid context file` but using this in package.json rather than the `--context gitlab.json` CLI arg:

```json
{
  "standard-version": {
    "context": "gitlab.json"
  }
}
```

Reply:
Looks good.
I'll change this PR ASAP.